src/engine/sidechain/enginerecord.cpp

```diff
@@ -249,6 +251,12 @@ void EngineRecord::writeCueLine() {
                             .arg(m_pCurrentTrack->getArtist())
                             .toUtf8());
 
+    if (m_bCueUsesFileAnnotation) {
+        m_cueFile.write(QString("    FILE \"%1\"\n")
```

Comment:
I believe you might be able to use a `QStringLiteral`

Reply:
For this one, fyi all the `m_cueFile.write` lines are existing code and use QString, I just added the if statement on line 254. I assumed whoever wrote that code used QString for a reason but idk.

Reply:
`QStringLiteral` is a `QString` :) ([doc](https://doc.qt.io/qt-6/qstring.html#QStringLiteral))

The value of a `QStringLiteral` is that it will create the `QString` at compilte time, instead of instantiating at every function calls. This is a macro optimisation.

The TL;DR is: if you use a static string in your `QString` (like here with `" FILE \"%1\"\n"`), you can make it a `QStringLiteral`.
You are correct that the one above could also be using `QStringLiteral` tho! Feel free to update them.

Reply:
Thanks for the explanation, I will update mine and the ones above.